binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -60,18 +60,16 @@
  * @author Oliver Libutzki
  * @author Kai Kreuzer - stabilized code
  * @author Andre Fuechsel - implemented switch off when brightness == 0
+ * @author Thomas Höfer - added thing properties
  *
  */
 public class HueLightHandler extends BaseThingHandler implements LightStatusListener {
 
-	public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES = Sets
-			.newHashSet(THING_TYPE_LCT001, THING_TYPE_LCT002,
-					THING_TYPE_LCT003, THING_TYPE_LLC001, THING_TYPE_LLC006,
-					THING_TYPE_LLC007, THING_TYPE_LLC010, THING_TYPE_LLC011,
-					THING_TYPE_LLC012, THING_TYPE_LLC013, THING_TYPE_LWL001,
-					THING_TYPE_LST001, THING_TYPE_LCT003, THING_TYPE_LWB004,
-					THING_TYPE_CLASSIC_A60_RGBW, THING_TYPE_SURFACE_LIGHT_TW,
-					THING_TYPE_ZLL_LIGHT);
+    public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES = Sets.newHashSet(THING_TYPE_LCT001, THING_TYPE_LCT002,
+            THING_TYPE_LCT003, THING_TYPE_LLC001, THING_TYPE_LLC006, THING_TYPE_LLC007, THING_TYPE_LLC010,
+            THING_TYPE_LLC011, THING_TYPE_LLC012, THING_TYPE_LLC013, THING_TYPE_LWL001, THING_TYPE_LST001,
+            THING_TYPE_LCT003, THING_TYPE_LWB004, THING_TYPE_CLASSIC_A60_RGBW, THING_TYPE_SURFACE_LIGHT_TW,
+            THING_TYPE_ZLL_LIGHT);
```

Comment:
For me the formatter is also fine... I understand the argument of Dennis, too... But at this concrete place it is difficult to check if all required thing types are part of the hash set... Maybe this is the reason why the original file was not formatted correctly...